benchmark/utils/main.swift

```diff
@@ -400,4 +410,12 @@ registerBenchmark(Walsh)
 registerBenchmark(WordCount)
 registerBenchmark(XorLoop)
 
-main()
+if #available(macOS 9999, *) {
+  
+  Task.runDetached {
```

Comment:
`Task.runDetached` is now `detach`

Reply:
Unfortunately detach was incorrectly marked not static in the compiler I'm using